src/interfaces.ts

```diff
@@ -153,6 +153,16 @@ export interface PrecomputedFlag {
   doLog: boolean;
 }
 
+export interface IPrecomputedBandit {
+  action: string | null;
+  variation: string;
+  actionProbability: number;
+  optimalityGap: number;
+  modelVersion: string;
+  actionAttributes: ContextAttributes;
```

Comment:
major: this is two fields in the spec to align with our bandit event forms
```suggestion
  actionNumericAttributes: Record<string, string>;
  actionCategoricalAttributes: Record<string, string>;
```
note that numeric attributes are converted to string when obfuscated.

(I'm pretty open to changing the format though to make it use nested object as that simplifies implementation in eppo core as well. But we need to make this change centrally.)

Reply:
I've changed to match the typing here. Created Obfuscated* versions of the type tree in order to capture the string nature of these attributes.